src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Triggers/EventHubAsyncCollector.cs

```diff
@@ -171,7 +182,7 @@ public async Task FlushAsync(CancellationToken cancellationToken = default(Cance
 
                 if (batch.Length > 0)
                 {
-                    await _parent.SendBatchAsync(batch);
+                    await _parent.SendBatchAsync(batch, _partitionKey);
```

Comment:
If we are using the same hashed paritionkey, is this guaranteed to be sent to the same eventub partition? I do not know if we are using the same EventHub client/connection or not. Plus if this will effect the partitionkey mapping.

One more question if the user created more than 32 keys. are we going to hash two values to the same partition?